change/@microsoft-fast-foundation-9eccb063-3575-4f30-8b53-d4fb397dcaf7.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "applying package updates",
```

Comment:
Should the change files for fast-foundation, fast-react-wrapper, and fast-router be here?